config/routes.rb

```diff
@@ -5,6 +5,8 @@
 
   resources :articles, only: [:index, :show]
 
+  resources :categories, only: [:show]
```

Comment:
Any reason why you are adding routes in a styling pr?

Reply:
That is out of the scope of this PT story and you are not writing any tests for it.

Reply:
we have now written tests for this